src/NServiceBus.AcceptanceTests/ConfigureEndpointLearningPersistence.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.AcceptanceTesting.Support;
+using NServiceBus.Persistence;
+
+public class ConfigureEndpointLearningPersistence : IConfigureEndpointTestExecution
+{
+    public Task Configure(string endpointName, EndpointConfiguration configuration, RunSettings settings, PublisherMetadata publisherMetadata)
+    {
+        storageDir = Path.Combine(@"c:\temp", Guid.NewGuid().ToString().Replace("-", "")); //can't use bindir since that will be to long on the build agents
```

Comment:
Bindir will fail on TC since the nunit runner has a very deep bin dir (ask me how I know)